Arriba/Arriba.Web/parts/QueryStats.jsx

```diff
@@ -10,8 +10,8 @@ export default React.createClass({
 
         // Write details for selected table
         var selectedDetails = [];
-        if (this.props.selectedData && this.props.selectedData.content) {
-            var selectedContent = this.props.selectedData.content;
+        if (this.props.selectedData) {
+            var selectedContent = this.props.selectedData;
```

Comment:
Rename variable since it's not unwrapping content anymore? Though I guess the caller does.

Reply:
I was actually wondering why it was called "selectedData" when the outer component calls the same object "listingData".

Reply:
Yeah, we should rename to listingData. I think the name was because it was for the "selected" table, but the name makes it conflict with the detail item data.